### ISSUE TITLE
 Aeternity: scalable smart contracts + oracle machine 

### ISSUE THREAD

**Issue by manelomanu** [francep2p/community#59]
Manel Ruiz, lead ambassador of Aeternity in Spain will explain how the Aeternity Blockchain works and what are its differences with the other smart contract platforms in the market. It will present the Starfleet program (next edition in 2020), the incubation program to create Dapps in Aeternity in which 8 teams have already received $ 100,000 in AE tokens and a six-month mentoring program with some of the best programmers in Europe. The attendees of the talk will be given a link so they can participate in remunerated online courses (100$) to learn how to program Smart contracts in the functional language Sophia."

Aeternity: scalable smart contracts + oracle machine

#### Languages

- [* ] 🇫🇷
- [ *] 🇬🇧

#### Estimated duration
30 min

every crytpo enthusiast is welcome! some part of my talk will be a bit more directed to developers and I'll invite them to do our remunerated online course(100$ in AE token) to learn the functional programming language for smart contract that we developed, Sophia.
4min- Presentation
8min - Scaling solutions
8min - Oracles
4min - mining Aeternity
6min - Starfleet program (incubation program to develop Apps in Aeternity and remunerated courses)
#### Required knowledge level

  
| Network      | Cryptography | Blockchain   | General Tech |
| ------------ | ------------ | ------------ | ------------ |
| [ ] None     | [ ] None     | [ ] None     | [ ] None     |
| [* ] Beginner | [* ] Beginner | [* ] Beginner | [* ] Beginner |
| [ ] Advanced | [ ] Advanced | [ ] Advanced | [ ] Advanced |
| [ ] Expert   | [ ] Expert   | [ ] Expert   | [ ] Expert   |

#### Video recording

- [* ] Yes (mandatory) 
- [* ] Yes (if possible) 
- [ ] No 
 I normally record myself with my professional camera but as you want
Presentation
Scaling solutions: Bitcoin NG and State Channels
Oracles
Mining Aeternity
Starfleet program (incubation program to develop Apps in Aeternity and remunerated courses)
Networking: Food and drinks(paid by us), airdrop of 2 Euros to each person in the audience and Aeternity t-shirts and merchandising

## Description of your talk

- 4min presentation
- 8min scaling solutions: Bitcoin NG and state channels
- 8 min oracles
- 4min  Cukko Cycle proof of work concensus algorithm
- 6min Starfleet program (incubation program to create dapps in aeternity in which already 8 teams have got $100.000 in AE token to develop a Dapp + our remunerated online courses($100) to learn the basics of our smart contract programing language, Sophia 
---

## About you

* Name:Manel Ruiz 
* Matrix handle: **TODO** 
* Discord handle: Manel o Manu #5954 
* Other contact info:manelomanu@protonmail.com
* Links to your previous talks: 
Aeternity Talks
Madrid
https://twitter.com/manelomanu/status/1132594994420883456?s=09
Zaragoza
https://empresason.com/not/6228/manel-ruiz-los-state-channels-permiten-transacciones-infinitas-a-cero-coste-/ EmpresasON el digital económico de Aragón https://empresason.com/not/6228/man...
Alicante
https://twitter.com/Ucoin_/status/1124750055439781893?s=19
Murcia
https://congresoblockchainmurcia.es/ 
* Other (your projects, etc...):I represented Bitlattice in the Startup panel of the European Blockchain convention 2018 and I'm a teacher in the Blockchain Institute of Barcelona 
* Availability (monthly number, date): I ll be in France from the 24th to the 7th of February, also managing a meetup in Paris that seems it will be on the 5th of February ;), so what about the wednesday 4th of March?

**Comment by moul** [comment]
@manelomanu can you confirm me that this issue is still open because you plan to make your presentation in other francep2p chapters i.e., lyon?

**Comment by manelomanu** [comment]
Yes I really liked yesterday meetup and would love to do one in Lyon Next month.

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by moul** [comment]
ok :+1: cc @lpieri

**Comment by manelomanu** [comment]
Guys, I don't see ant post about the Monthly meetup in Lyon on the 4th of March...is still Up?

Thanks!

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by gorhgorh** [comment]
Hello @manelomanu, I think Lyon's March meetup is canceled, can you confirm @lpieri ?

**Comment by lpieri** [comment]
Hello @manelomanu, the next Lyon P2P will be the 12 March 😄

**Comment by manelomanu** [comment]
Hey, great! I would like to do s presentation of Aeternity please

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by gorhgorh** [comment]
@lpieri my mistake then :)

**Comment by lpieri** [comment]
@manelomanu the presentation of Aeternity is planned :)

**Comment by manelomanu** [comment]
Ahh ok

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by manelomanu** [comment]
Can you send me the link of the event please? I don't see It in the P2P meetup Group

Thanks!

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by lpieri** [comment]
Yes, here are the link: https://www.meetup.com/fr-FR/France-P2P/events/269002377/ & https://github.com/francep2p/community/issues/74

**Comment by manelomanu** [comment]
Thanks Louise!!😁

Here is the Crypto Lyon Telegram Group, just in case you want to join It and promote the events of France P2P
I m going to buy the tickets to come to Lyon on the 12th. Can you please confirm me that We are going to do the event?

Thanks

Manel

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by manelomanu** [comment]
Sorry here is the Telegram link

Crypto Lyon
La communauté lyonnaise des Crypto actifs et technologies Blockchain

https://t.me/cryptolyonforum

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by manelomanu** [comment]
Hallo Louise,

j ai dejá parlez avec Jimmy du Group Crypto Lyon, il ma dit que vous vous connez dejà, desolé pour mon derrier email.

Je vous partage un petit text pour la petite presentation que je vais a faire, peut être il peut
Être interesant de le metre dans la info du meetup

Manel Ruiz(Blockchain Institute of Barcelona), lead ambassador d'Aeternity à Espagne, présentera les avancées de la plateforme de contrats intelligents depuis le lancement de son mainnet l'année dernière.Il expliquera les fonctionnalités d'Aeternity qui en font une Blockchain unique où les contrats intelligents scalables interagissent avec des informations du monde réel via des oracles décentralisés.

Il présentera le programme Starfleet, le programme d'incubation pour créer Dapps in Aeternity où plus de 25 équipes ont déjà obtenu un financement de 100000 $ en AE + un programme de mentorat de 6 mois avec certains des meilleurs développeurs en Europe. Il donnera un lien pour participer aux cours payants d'Aeternity (75 $) pour apprendre à utiliser le langage de programmation fonctionnel "Sophia".

  Il y aura un 2 Euro Airdrop pour les participants au meetup pour télécharger la base Daepp et essayer le Aeterntiy Naming System.

Merci
:-)

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by lpieri** [comment]
Hello Manel,

Oui je suis déjà en contact avec CryptoLyon, il faudrait que je les contacts ^^
Merci pour la description, je vais m'en servirais pour les annonces de l'évent 😄

**Comment by manelomanu** [comment]
Bonjour,

Es qu'on peux faire quelque chose pour avoir plus convocatoire et attendants le jeudi?

Manel

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by manelomanu** [comment]
Hello Louise,

Ça va pour le meetup de demain?

Je vous seulment 6 persones qui vont a attendre le meetup.

Es que il y aurà des autres?

Merci

Manel

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by lpieri** [comment]
Hello Manel,

Le meetup de demain est toujours maintenu, je pense par contre que il n'y auras pas grand monde avec le covid-19 :(

**Comment by manelomanu** [comment]
Hallo,

Tú as partagé dejà dans le Group Crypto Lyon?

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by lpieri** [comment]
Oui :)

**Comment by manelomanu** [comment]
Combien de personnes on confirmé assitance?

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by manelomanu** [comment]
I'm just not 100% sure to come from Barcelona in case we are less than 10. It's a very cheap flight but this Coronavirus thing sucks, you know

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by lpieri** [comment]
Okay, I get it. I think he'll have less than 10 people at the event. Are you okay with postponing the event till next month?

**Comment by manelomanu** [comment]
I'm not sure about what is going to be the situation in one month, I Will like to come, but need to make sure

Enviado desde ProtonMail móvil

-------- Mensaje original --------
